docs/pages/admin-guides/access-controls/rbac/getting-started-rbac.mdx

```diff
@@ -0,0 +1,315 @@
+---
+title: Getting Started With Role-Based Access Control
+description: Get started using Teleport Role-Based Access Control.
+---
+
+In Teleport, any local, SSO, or robot user can be assigned one or several roles. Roles govern access to databases, SSH servers, Kubernetes clusters, Windows desktops, and web apps.
```

Comment:
```suggestion
In Teleport, any local, SSO, or bot user can be assigned one or several roles. Roles govern access to databases, SSH servers, Kubernetes clusters, Windows desktops, and web apps.
```
In other docs pages, we say "bot user" instead of "robot user".

Reply:
Roles also configure the ability to manage a Teleport cluster (the `rules:` section of a role).

Reply:
Fixed!